src/components/RecipesDisplay/RecipeBrief.jsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import { nanoid } from 'nanoid';
 
+import './RecipeBrief.css';
+
 export default function RecipesBrief({ recipes }) {
   const data = recipes && recipes.map((recipe) => {
     const { name, thumbnail_url: thumbnailUrl } = recipe;
     return (
-      <div key={nanoid()}>
-        <img src={thumbnailUrl} />
-        <h1>{name}</h1>
-        {/* Link for Details */}
+      <div key={nanoid()} className="recipe-card bg-no-repeat bg-cover bg-center rounded-lg flex flex-col items-start justify-end" style={{ backgroundImage: `url(${thumbnailUrl})` }}>
```

Comment:
I assume nanoid is a random key generator. It is not recommended to use a random key generator as a key. You should use some sort of combination of strings from the recipes object or a unique id provided by the backend, which should also be in the recipes object.

Reply:
updated the key with the id from the data as recommended.